### PR TITLE
#2045 remove this check

### LINKED ISSUE #2045: Reuse code in tools.throttle and tools.eventsBuffer
## Type of report

Task

## Provide description of the task

In #1993 we added throttling function, however it introduced some copy'n'pasted code as mentioned in review comment https://github.com/ckeditor/ckeditor-dev/pull/1996#pullrequestreview-124055850.

The code should be reused.

## Other details

* Browser: Any
* OS: Any
* CKEditor version: 4.10.0

### PR DESCRIPTION
**Pull request by devanecondition**
partially fixes #2045

### CODE REVIEW

File: plugins/confighelper/plugin.js [hunk 1]
```diff
@@ -16,9 +16,6 @@ function dataIsEmpty( data )
 	if ( !data)
 		return true;
 
-	if ( data.length > 20 )
-		return false;
-
 	var value = data.replace( /[\n|\t]*/g, '' ).toLowerCase();
```
Comment (nodesocket) [inline plugins/confighelper/plugin.js]: Please can we refactor this?

```javascript
var _ = require('lodash');
var matches = ['<br>', '<p>&nbsp;<br></p>', '<p><br></p>', '<p>&nbsp;</p>', '&nbsp;', ' ', '&nbsp;<br>', ' <br>', '<p class="fs-14"><br></p>', '<p class="fs-14">&nbsp;<br></p>'];
if(!value || _.includes(matches, value) {
   return true;
}
```

File: plugins/confighelper/plugin.js [hunk 2]
```diff
@@ -16,9 +16,6 @@ function dataIsEmpty( data )
...
 	if ( !value || value == '<br>' || value == '<p>&nbsp;<br></p>' || value == '<p><br></p>' || value == '<p>&nbsp;</p>' || value == '&nbsp;' || value == ' ' || value == '&nbsp;<br>' || value == ' <br>' || value == '<p class="fs-14"><br></p>' || value == '<p class="fs-14">&nbsp;<br></p>')
```
Comment (nodesocket) [inline plugins/confighelper/plugin.js:20]: Please can we refactor this?

``` javascript
var _ = require('lodash');
var matches = ['<br>', '<p>&nbsp;<br></p>', '<p><br></p>', '<p>&nbsp;</p>', '&nbsp;', ' ', '&nbsp;<br>', ' <br>', '<p class="fs-14"><br></p>', '<p class="fs-14">&nbsp;<br></p>'];
if(!value || _.includes(matches, value) {
   return true;
}
```

Comment (acusti) [inline plugins/confighelper/plugin.js:20]: @nodesocket We’re trying to limit our changes. This is the ckeditor source, which, sadly, we’ve had to fork, but we still want to maintain the minimal surface area of local modifications.

Comment (nodesocket) [inline plugins/confighelper/plugin.js:20]: Sure. Though their code makes me throw up a little in my mouth :mask: . Shouldn't it also be catching `<br />`?

Comment (dok) [inline plugins/confighelper/plugin.js:20]: It's actually community driven code. This plugin isn't something they have written.

**Comment by dok** [comment]
LGTM

**Pull request merged**